auditlog/models/rule.py

```diff
@@ -343,6 +363,12 @@ def read(self, fields=None, load="_classic_read", **kwargs):
                 return result
             self = self.with_context(auditlog_disabled=True)
             rule_model = self.env["auditlog.rule"]
+            if self.env.user in users_to_exclude_ids:
+                return result
+            for d in fields:
+                if d not in fields_to_exclude_ids:
+                    break
+                return result
```

Comment:
hmmm this will return out of read on the very first iteration. What are you trying to achieve?